.github/workflows/plugins-ci.yml

```diff
@@ -120,12 +125,47 @@ jobs:
       - name: Run tests
         run: npm test
 
+  fastify-integration:
+    name: Test Fastify Integration
+    runs-on: ubuntu-latest
+    if: >
+      !failure() &&
+      !cancelled() &&
+      inputs.fastify-integration == true
+    permissions:
+      contents: read
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Install dependencies
+        run: npm i --ignore-scripts
+
+      - name: Test Fastify Integration
+        run: |
+          npm link &&
+          PACKAGE_NAME=$(node -p "require('./package.json').name")
+          git clone https://github.com/fastify/fastify.git --depth 1 &&
+          cd fastify &&
+          npm i --ignore-scripts &&
+          npm link $PACKAGE_NAME &&
+          npm run unit
+
   automerge:
     name: Automerge Dependabot PRs
     if: >
         github.event_name == 'pull_request' &&
         github.event.pull_request.user.login == 'dependabot[bot]'
-    needs: test
+    needs: 
+      - test
+      - fastify-integration
```

Comment:
is `fastify-integration` considered done when we have `inputs.fastify-integration = false`?